WordPress/Classes/ViewRelated/Site Creation/Web Address/AddressTableViewCell.swift

```diff
@@ -0,0 +1,317 @@
+import UIKit
+import WordPressKit
+
+final class AddressTableViewCell: UITableViewCell {
+
+    // MARK: - Dependencies
+
+    private let domainPurchasingEnabled = FeatureFlag.siteCreationDomainPurchasing.enabled
+
+    // MARK: - Views
+
+    private var borders = [UIView]()
+
+    private let domainLabel: UILabel = {
+        let label = UILabel()
+        label.adjustsFontForContentSizeCategory = true
+        label.font = Appearance.domainFont
+        label.textColor = Appearance.domainTextColor
+        return label
+    }()
+
+    private let tagsLabel: UILabel = {
+        let label = UILabel()
+        label.adjustsFontForContentSizeCategory = true
+        label.font = Appearance.tagFont
+        return label
+    }()
+
+    private let dotView: UIView = {
+        let size = Appearance.dotViewRadius * 2
+        let view = UIView()
+        view.clipsToBounds = true
+        view.layer.cornerRadius = Appearance.dotViewRadius
+        view.translatesAutoresizingMaskIntoConstraints = true
+        view.frame.size = .init(width: size, height: size)
+        return view
+    }()
+
+    private let checkmarkImageView: UIView = {
+        let configuration = UIImage.SymbolConfiguration(weight: .semibold)
+        let image = UIImage(systemName: "checkmark", withConfiguration: configuration)
+        let view = UIImageView()
+        view.isHidden = true
+        view.image = image
+        view.translatesAutoresizingMaskIntoConstraints = true
+        view.sizeToFit()
+        return view
+    }()
+
+    private let costLabel: UILabel = {
+        let label = UILabel()
+        label.adjustsFontForContentSizeCategory = true
+        label.font = Appearance.domainFont
+        label.textColor = Appearance.domainTextColor
+        label.numberOfLines = 2
+        return label
+    }()
+
+    // MARK: - Init
+
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        commonInit()
+        if domainPurchasingEnabled {
+            setupSubviews()
+        }
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        commonInit()
+    }
+
+    private func commonInit() {
+        self.accessibilityTraits = .button
+        self.accessibilityHint = NSLocalizedString(
+            "Selects this domain to use for your site.",
+            comment: "Accessibility hint for a domain in the Site Creation domains list."
```

Comment:
Just highlighting the missing "key". If it is a copy over or anything, you can ignore.

Reply:
Indeed, the key is missing but that's an old localized string that I didn't want to change.